docs/sources/setup/provisioning.md

```diff
@@ -55,6 +55,7 @@ If you need an advanced version of the datasource, use the following format:
     tlsAuthWithCACert: <<true or false>> -- false by default
     serverName: <<server name that matches in certificate for tlsAuthWithCACert>>
     timeoutInSeconds: <<60>> -- or whatever the timeout you want set. If not set defaults to 60.
+    allowNonGetPostMethods: false
```

Comment:
What about calling it `allowDestructingHTTPMethods` or something more scary to make sure admins really think this through. I was at a talk about developer UX and one of the things that stuck with me is how speaker praised how react team called `dangerouslySetInnerHTML` method, because you can still do it, but it makes it clear that bad things might happen and you should be very aware of that possibility. And this seems like a similar thing to me.

Reply:
What about `allowDangerousHTTPMethods`? Or maybe we could check with technical writers or UX or product if they have suggestion 🤔